profile_bluesky/startup/instrument/devices/lambda_750k.py

```diff
@@ -477,8 +485,9 @@ def watch_acquire(value, old_value, **kwargs):
         time.sleep(0.005)  # wait for the shutter to move out of the way
         self.cam.state.subscribe(watch_state)
         self.imm1.capture.subscribe(watch_acquire)
-        for plugin in (self.imm0, self.imm1, self.imm2):
-            plugin.capture.put(1, wait=False)
+        # for plugin in (self.imm0, self.imm1, self.imm2):
+        #     plugin.capture.put(1, wait=False)
+        self.imm1.capture.put(1, wait=False)
```

Comment:
Is this line a duplicate of the one below?  Can it be removed?  Otherwise comment why it is needed twice.